app/main.py

```diff
@@ -1,6 +1,25 @@
+import requests
+import os
+
+
+WEATHER_SITE = "https://api.weatherapi.com"
+API_KEY = os.environ["weatherapi_key"]
+URL = f"{WEATHER_SITE}/v1/current.json?key={API_KEY}" # noqa
+
+FILTERING = "Paris" # noqa
```

Comment:
why do you use # noqa?
is there a problem with linter?
Don't use the redundant variable WEATHER_SITE

Reply:
I used # noqa because I had some problems with flake8